teacher.rb

```diff
@@ -11,5 +11,5 @@ def can_use_services?
   end
 end
 
-teacher = Teacher.new(36, 'mathematics', 'Olivier')
+teacher = Teacher.new(26, 'Web Development', 'Abiodun')
 p teacher.can_use_services?
```

Comment:
- [ ] [OPTIONAL] Please remove any unnecessary code to make your code cleaner and more professional.